toml.md

```diff
@@ -109,6 +109,12 @@ may contain ASCII digits, dashes and underscores. Punctuation, spaces, arrows, b
 and private use characters are not allowed.  Note that bare keys are allowed to be
 composed of only ASCII digits, e.g. `1234`, but are always interpreted as strings.
 
+* From the ASCII characters, only A-Z, a-z, 0-9, _ and - are allowed
+* From the rest of the first 256 characters, only 0080-00BF, "×" (00D7) and "÷" (00F7) are disallowed
```

Comment:
I would use the U+ syntax here, as elsewhere in the spec ("U+0080 to U+00BF" etc.)

Reply:
was thinking of that too, will do ;)

Reply:
Done